### DISCUSSION TITLE
SQLite Library

### DISCUSSION THREAD

**Discussion by lildinti** [Sketchware-Pro/Sketchware-Pro#621] [Q&A] [answered]
I found and used this in kodular. how can I use it  in sketchware? i can modify the code. 
I'd like to create blocks for it like the extension in kodular without having tou add code directly. 
Cheers,
Lildinti

https://github.com/frdfsnlght/aix-SQLite

**Comment by PranavPurwar** [comment]
add this in java/kotlin manager https://github.com/frdfsnlght/aix-SQLite/blob/master/src/SQLite.java

**Comment by PranavPurwar** [comment]
you would also need to add appinvent dependency to your app

**Comment by lildinti** [comment]
Thanks for the reply, Pranav.
 How do I add dependency? Any tutorial somewhere?

Thanks heaps

**Reply by PranavPurwar** [reply, marked as answer]
with library manager
